src/main/java/seedu/address/model/person/ClientStatus.java

```diff
@@ -11,7 +11,7 @@ public class ClientStatus {
 
     public static final String MESSAGE_CONSTRAINTS =
             "Client status must be either “active”, “unresponsive”,"
-            + " “potential” or “old” (case insensitive).";
+            + " “potential”, “old” or “blacklisted” (case insensitive).";
```

Comment:
Ignore this change, its the same as #246 for some reason my branching has gone to shit